app/src/main/java/fr/gaulupeau/apps/Poche/ui/ReadArticleActivity.java

```diff
@@ -785,6 +787,17 @@ public void onClick(DialogInterface dialog, int which) {
                             case 1:
                                 ServiceHelper.addLink(ReadArticleActivity.this, url);
                                 break;
+                            case 2:
+                                ClipboardManager clipboard = (ClipboardManager) getSystemService(Context.CLIPBOARD_SERVICE);
+                                ClipData clip = ClipData.newRawUri("wallabag", Uri.parse(url));
+                                clipboard.setPrimaryClip(clip);
+                                break;
+                            case 3:
+                                Intent sendIntent = new Intent();
+                                sendIntent.setAction(Intent.ACTION_SEND);
+                                sendIntent.putExtra(Intent.EXTRA_TEXT, url);
+                                sendIntent.setType("text/plan");
+                                startActivity(sendIntent);
```

Comment:
Please add `break;` (it will be useful if someone adds another case without checking).

Reply:
Thank you. I fixed this.